lighthouse-core/gather/gatherers/script-elements.js

```diff
@@ -8,8 +8,7 @@
 const Gatherer = require('./gatherer.js');
 const NetworkAnalyzer = require('../../lib/dependency-graph/simulator/network-analyzer.js');
 const NetworkRequest = require('../../lib/network-request.js');
-const getElementsInDocumentString = require('../../lib/page-functions.js').getElementsInDocumentString; // eslint-disable-line max-len
-const pageFunctions = require('../../lib/page-functions.js');
+const {getElementsInDocument, getNodeDetailsString} = require('../../lib/page-functions.js');
```

Comment:
Should we keep this consistent with the style of link-elements?

Reply:
> Should we keep this consistent with the style of link-elements?

yeah, which is the way that breaks when minified? If we're getting confused we should definitely document this :)

Reply:
Yes, and it is necessary, due to mangling #11463

Reply:
> Yes, and it is necessary, due to mangling #11463

this is unfortunate, since we lose all `deps` type checking benefits then :/

(see also https://github.com/GoogleChrome/lighthouse/pull/11463#discussion_r497088597 for an example of the mangling)

Reply:
yea, reminder tho that we could get it  back if we namespace the page functions (idea 3 https://github.com/GoogleChrome/lighthouse/pull/10781 )